lib/riffed/client.ex

```diff
@@ -207,12 +223,17 @@ defmodule Riffed.Client do
           {:error, :closed} ->
             new_client = Client.reconnect(client)
             call_thrift(new_client, call_name, args, retry_count + 1)
-          err = {:error, _} ->
-            {new_client, err}
+
+          err = {:error, reason} ->
+            # try to close this, this process is going to die anyway.
```

Comment:
Yes, this is the change. Before we would handle them in a non-crashy way, but now we crash on all non-closed errors. 

I can take you through why we're going to do this, and I need to prove it out in production.